src/utils/input.js

```diff
@@ -35,3 +40,26 @@ export const makeInputTypeAndLength = typeDefinition => {
 
   return result
 }
+
+/**
+ * @param {number} valueLength - Length of input value
+ * @param {boolean} isRequired - If value is required
+ * @param {number} inputMaxLength - Minimum limit of length of the input value
+ * @returns {boolean}
+ */
+export const checkInputConstraints = (
```

Comment:
It seems a bit weird to pass the length and not the raw value here. The method is name `checkInput` but it is more `checkLength`.

Reply:
It doesn't only test the length, I can propose something more verbose like `checkConstraintsOfIinput`.